packages/flutter_tools/templates/plugin_shared/pubspec.yaml.tmpl

```diff
@@ -13,7 +13,7 @@ dependencies:
 {{#web}}
   flutter_web_plugins:
     sdk: flutter
-  web: ^0.5.1
+  web: '>=0.5.1 <2.0.0'
```

Comment:
Is there a particular reason to still allow versions before 1.0? Since these templates are for new projects, I think it's fine to raise it so the project can safely use the newer APIs.

Reply:
It was out of caution. The app developer might expect this plugin to be compatible with other packages which still use web < 1.0. But I'm totally fine raising the version to ^1.0.0. I guess when this change is getting to stable and also the potential plugin is released, every package should have been upgraded to use 1.0.0 at least.